backend/main.py

```
@@ -0,0 +1,88 @@
+from fastapi import FastAPI, HTTPException, Depends, status
```

Ruff (0.12.2)

`fastapi.HTTPException` imported but unused

Remove unused import

(F401)